---a/net/minecraft/block/BlockSoulSand.java
+++b/net/minecraft/block/BlockSoulSand.java
@@ -8,26 +8,23 @@
 import net.minecraft.util.BlockPos;
 import net.minecraft.world.World;
 
-public class BlockSoulSand extends Block
-{
+public class BlockSoulSand extends Block {
+
     private static final String __OBFID = "CL_00000310";
 
-    public BlockSoulSand()
-    {
+    public BlockSoulSand() {
         super(Material.sand);
         this.setCreativeTab(CreativeTabs.tabBlock);
     }
 
-    public AxisAlignedBB getCollisionBoundingBox(World worldIn, BlockPos pos, IBlockState state)
-    {
+    public AxisAlignedBB getCollisionBoundingBox(World worldIn, BlockPos pos, IBlockState state) {
         float f = 0.125F;
-        return new AxisAlignedBB((double)pos.getX(), (double)pos.getY(), (double)pos.getZ(), (double)(pos.getX() + 1), (double)((float)(pos.getY() + 1) - f), (double)(pos.getZ() + 1));
+        return new AxisAlignedBB((double) pos.getX(), (double) pos.getY(), (double) pos.getZ(), (double) (pos.getX() + 1), (double) ((float) (pos.getY() + 1) - f), (double) (pos.getZ() + 1));
     }
 
-    public void onEntityCollidedWithBlock(World worldIn, BlockPos pos, IBlockState state, Entity entityIn)
-    {
+    public void onEntityCollidedWithBlock(World worldIn, BlockPos pos, IBlockState state, Entity entityIn) {
         entityIn.motionX *= 0.4D;
         entityIn.motionZ *= 0.4D;
     }
-}
+}
